src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -199,6 +200,8 @@ protected void onCreate(Bundle savedInstanceState) {
         }   // else, Fragment already created and retained across configuration change
 
         Log_OC.v(TAG, "onCreate() end");
+
+        AppRater.appLaunched(this, getApplicationContext().getPackageName());
```

Comment:
I think you don't need here the application context since you can use the activity context, so just `getPackageName()` should work